### PR TITLE
Finish PackageReference migration

### PR DESCRIPTION
**Pull request by mhutch**

### CODE REVIEW

File: Test/UnitTests/UnitTests.csproj [hunk 1]
```diff
@@ -26,13 +26,8 @@
     <Reference Include="System" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
-    <PackageReference Include="NUnit">
-      <Version>3.9.0</Version>
-    </PackageReference>
-    <PackageReference Include="NUnit3TestAdapter">
-      <Version>3.10.0</Version>
-    </PackageReference>
-    <PackageReference Include="Mono.Cecil" Version="0.10.0" />
```
Comment (KirillOsenkov) [inline Test/UnitTests/UnitTests.csproj]: Cecil not required?

Comment (KirillOsenkov) [reply Test/UnitTests/UnitTests.csproj]: Yeah doesn't seem to be needed:
```
C:\xmd\main\external\mono-addins\Test\UnitTests>refdump C:\xmd\main\external\mono-addins\Test\UnitTests\bin\Debug\UnitTests.dll
UnitTests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
References:
Mono.Addins, Version=1.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
Mono.Addins.Setup, Version=1.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
nunit.framework, Version=3.9.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb
System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089```

Comment (Therzok) [reply Test/UnitTests/UnitTests.csproj]: It should be there, since we depend on CecilReflector.

Comment (Therzok) [reply Test/UnitTests/UnitTests.csproj]: Maybe CecilReflector imports it transitively.

**Pull request merged**